Tools/Scripts/webkitpy/port/darwin.py

```diff
@@ -247,6 +247,9 @@ def _path_to_image_diff(self):
 
         return _image_diff_in_build_path
 
+    def path_to_api_test_binaries(self):
+        return {binary: self._build_path(binary) for binary in super(DarwinPort, self).API_TEST_BINARY_NAMES + ['TestIPC']}
+
```

Comment:
nit: For legibility, I would prefer just overriding API_TEST_BINARY_NAMES instead of inheriting like this. We already do it for other ports.
```suggestion
    API_TEST_BINARY_NAMES = ['TestWTF', 'TestWebKitAPI', 'TestIPC']

```